core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -46,19 +46,34 @@ CFG_WITH_STATS ?= y
 
 ifeq ($(PLATFORM_FLAVOR),fvp)
 CFG_TEE_CORE_NB_CORE = 8
+CFG_TZDRAM_START ?= 0x06000000
```

Comment:
It would be easier to read the values if they where aligned like:
```
CFG_TZDRAM_START ?= 0x06000000
CFG_TZDRAM_SIZE  ?= 0x02000000
CFG_SHMEM_START  ?= 0x83000000
CFG_SHMEM_SIZE   ?= 0x00200000
```

This is a matter of taste so I leave it to you to decide.